src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__content {
+    padding-inline: 16px;
+  }
+
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
```

Comment:
It's a good practice to group related CSS properties together. For example, font properties can be grouped together (font-size, font-weight, line-height, color).